h2o-core/src/main/java/hex/ModelMetrics.java

```diff
@@ -370,7 +370,7 @@ public int compare(Integer idx1, Integer idx2) {
             new String[rel_imp.length][], sorted_imp);
   }
 
-  private static Key<ModelMetrics> buildKey(Key model_key, long model_checksum, Key frame_key, long frame_checksum) {
+  public static Key<ModelMetrics> buildKey(Key model_key, long model_checksum, Key frame_key, long frame_checksum) {
```

Comment:
This method should ideally stay private

Reply:
I was thinking it should, but it appears it is needed at the top of the stack. Could reflection be used?

Reply:
We can avoid it by using the suggestion with "withModelAndFrame", then we can keep it private.

Reply:
@michalkurka @navdeep-G These are good suggestions.  I can clean this up in the fix release if that's okay.